browser/src/canvas/CanvasSectionProps.js

```diff
@@ -26,32 +34,32 @@ L.CSections.Tiles = 				{ name: 'tiles'				, zIndex: 5 };
 L.CSections.Overlays =				{ name: 'overlay'			, zIndex: 5 };
 L.CSections.CalcGrid = 				{ name: 'calc grid'			, zIndex: 5 };
 L.CSections.Debug.Splits = 			{ name: 'splits'			, zIndex: 5 };
-L.CSections.Debug.TilePixelGrid = 	        { name: 'tile pixel grid'	, zIndex: 5 };
-L.CSections.Debug.PreloadMap = 	                { name: 'preload map'	        , zIndex: 5 };
-
+L.CSections.Debug.TilePixelGrid = 	{ name: 'tile pixel grid'	, zIndex: 5 };
```

Comment:
looks like alignment is worse now

Reply:
I think it looks like that here. In code editor, it's normal.